package.json

```diff
@@ -3,7 +3,7 @@
   "description": "React component for messenger checkbox plugin",
   "license": "MIT",
   "version": "0.1.0",
-  "main": "lib/index.js",
+  "main": "lib/MessengerCheckbox.js",
```

Comment:
@remiroyc You should not delete the `src/MessengerCheckbox.js` file and apply this line change at the same time. This will cause missing entry when installing the package.